tests/test_recipes.py

```diff
@@ -492,26 +492,28 @@ def test_first(self):
     def test_multiple(self):
         """ensure can catch multiple exceptions"""
 
-        class Fiz(Exception):
+        class FizzError(Exception):
```

Comment:
No.

In this case, adding `Error` to the name makes the code worse.  The exceptions are used as a named signal rather than an error casing.